tests/integration/features/metadata.feature

```diff
@@ -42,3 +42,25 @@ Feature: Metadata
     version: {{ version }}
     schema_only: true
     """
+
+  @require_version_22.3
+  Scenario: Check fail reason
+    Given we have executed queries on clickhouse01
+    """
+    CREATE TABLE foo (
+        id UInt32
+    ) 
+    ENGINE = MergeTree()
+    ORDER BY id;
+
+    INSERT INTO foo SELECT number FROM numbers(100);
+    """
+    Given we have executed command on clickhouse01
+    """
+    rm -rf /var/lib/clickhouse/data/default/foo/all_1_1_0
+    """
+    When we try to create clickhouse01 clickhouse backup
+    Then metadata of clickhouse01 backup #0 contains value for "fail_reason" which begins with
+    """
+    FileNotFoundError: [Errno 2] No such file or directory
+    """
```

Comment:
I'd expect that the field named `fail_reason` contains more high-level values: "disk quota exceeded", "memory limit exceeded", "internal error", etc.

If we just add an exception to metadata, suggest to rename the field to `exception`.